src/greyVest/Pager.js

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash/fp'
+import React from 'react'
+import GVPagerItem from './PagerItem'
+import GVIcon from './Icon'
+import Flex from './Flex'
+
+let Pager = ({
```

Comment:
temporary; this is already in grey-vest (and this whole folder is about to be deleted anyway)